### ISSUE TITLE
External markdown enhancements

### ISSUE THREAD

**Issue by StevenClontz** [pi-base/web#184]
- [x] - On `/dev/preview`, don't show the pi-Base markdown (duplicative of the textbook entry)
- [x] - Show the name of the space/property for `{S123}` and `{P123}` citations.
- [x] - Show the theorem for `{T123}` citations
- [x] - Support `{S123|P456}` traits.

**Comment by StevenClontz** [comment]
For the first, a hint: use this in `packages/viewer/src/components/Shared/Source.svelte` to add optional `internal` and `external` attributes to the Source component

```typescript
  export let internal = true
  export let external = true
```

**Comment by StevenClontz** [comment]
More hints:

```typescript
import context from '@/context'
const { spaces } = context()
```

```typescript
const name = $spaces.find(`S${id}`)?.name
```

**Comment by StevenClontz** [comment]
And `{S123|P123}` should link to `/spaces/S000123/properties/P000123`

**Comment by StevenClontz** [comment]
Closed by #193
